tools/mkto-poi/mkto-poi.js

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable no-underscore-dangle, import/no-unresolved */
+import getStyle from 'https://da.live/nx/utils/styles.js';
+import { LitElement, html } from 'da-lit';
+
+const POI_URL = 'https://milo.adobe.com/tools/marketo-options.json';
+
+async function getMarketoPOI() {
+  const resp = await fetch(POI_URL);
+  if (!resp.ok) {
+    const error = 'Error fetching marketo options';
+    const mktoError = new CustomEvent('mktoOptionsError', { detail: { message: error } });
+    document.dispatchEvent(mktoError);
+    return [];
+  }
+
+  const data = await resp.json();
+  return data;
+}
+
+function poiPair(data) {
+  const poiList = data.poi.data;
+  return poiList.reduce((rdx, item) => {
+    const { Key, Value } = item;
```

Comment:
Why are these capitalized?

Reply:
They are capitalized in the data coming from the sheet, so using capitals here to ensure I get the correct property from destructuring.